my-app/src/components/SearchBar.js

```diff
@@ -26,6 +26,7 @@ export default function SearchBar() {
       <p>BaseCheck wants to ensure that every person has the accessible opportunity to stay well-informed about the pandemic.</p>
       <div className="search-form">
         <Autocomplete
+          className="search-input"
```

Comment:
@AkolyVongdala What is the purpose of this class name? Just curious because I don't see any styling set for it in the css file

Reply:
Please remove this, it doesn't serve any purpose... I not sure when I added that...